ioos_qartod/qc_tests/qc.py

```diff
@@ -281,6 +282,19 @@ def attenuated_signal_check(arr, times, min_var_warn, min_var_fail,
     return flag_arr
 
 
+@add_qartod_ident(11, 'Current Direction')
+def current_direction_check(arr, sensor_min=0.0, sensor_max=360.0):
+    """
+    Ensure current direction values fall between 0 and 360 degrees.
+
+    """
+    flag_arr = np.ones_like(arr, dtype='uint8')
+
+    flag_arr[(arr < sensor_min) |
+             (arr > sensor_max)] = QCFlags.BAD_DATA
```

Comment:
@lukecampbell we can implement this a special case of `range_check()` the same way I did for #65. Let me know which style works best for your goals here.